src/crypto/crypto_context.cc

```diff
@@ -658,33 +665,53 @@ void SecureContext::SetEngineKey(const FunctionCallbackInfo<Value>& args) {
 }
 #endif  // !OPENSSL_NO_ENGINE
 
+bool SecureContext::AddCert(BIOPointer&& bio) {
+  if (!bio) return false;
+  cert_.reset();
+  issuer_.reset();
+
+  if (!SSL_CTX_use_certificate_chain(
+          ctx_.get(), std::forward<BIOPointer>(bio), &cert_, &issuer_)) {
```

Comment:
I was _tres_ confused here for a second but:

1. SSL_CTX_use_certificate_chain is a node thing, not openssl's, contrary to what the name suggests, right?

2. `std::forward` isn't necessary? Already takes a `BIOPointer&&`, doesn't it? Or does the compiler get confused by our _other_ SSL_CTX_use_certificate_chain?

We really should rename that function...

Reply:
Handled in #45912 (and yes, that function should be renamed)